static/app/utils/profiling/hooks/useResizableDrawer.tsx

```diff
@@ -1,47 +1,80 @@
 import {useCallback, useEffect, useRef, useState} from 'react';
 
-interface UseVerticallyResizableDrawerOptions {
-  initialHeight: number;
-  minHeight?: number;
+export interface UseResizableDrawerOptions {
+  direction: 'horizontal-ltr' | 'horizontal-rtl' | 'vertical';
```

Comment:
This is set because depending on the table placement, you are either moving the left or the right edge. Which means that distance that the mouse traveled should be inverted. See below where I multiply with -1